docs/internals/contributing/index.txt

```diff
@@ -4,7 +4,7 @@ Contributing to Django
 
 Django is a community that lives on its volunteers. As it keeps growing, we
 always need more people to help others. You can contribute in many ways, either
-on the framework itself or in the wider ecosystem.
+to the framework itself or in the wider ecosystem.
```

Comment:
This is also correct.

Reply:
Doesn't "contribute to" sounds more accurate than "contribute on"?

Reply:
"on" is correct IMO.

Reply:
ok. I will revert!